setup.py

```diff
@@ -37,6 +39,6 @@
              'scripts/plot_skymap.py',
              'scripts/primarybeammap_tant_test.py',
              'scripts/track_and_suppress.py'],
-    install_requires=["numpy", "astropy", "skyfield>=1.16", "matplotlib", "scipy>=0.15.1", "h5py"],
+    install_requires=["numpy>=1.20", "astropy", "skyfield>=1.16", "matplotlib", "scipy>=0.15.1", "h5py"],
```

Comment:
I'm actually just realising that in reality this `>=1.20` is not necessary.  Even several versions older `numpy` will be happy with `complex`.